### DISCUSSION TITLE
Remove hard ref to Json.net

### DISCUSSION THREAD

**Discussion by nhaberl** [jchristn/RestWrapper#5] [Ideas]
Hi,  would be great to abstract the serialization so any Json library can be used ... as there is native support now I won't rather install Json.Net.

**Comment by jchristn** [comment]
Hi @nhaberl I'm open to doing that (and could do that in other projects too). Could you point me toward a reference?  Thanks!

**Comment by jchristn** [comment]
Moving this to a discussion, not a bug.  If you have a PR on this I'd be happy to integrate if it aligns with the intent of the library.  Cheers, Joel
